tests/conftest.py

```diff
@@ -350,6 +350,13 @@ def signed_create_tx(b, create_tx):
     return create_tx.sign([b.me_private])
 
 
+@pytest.fixture
+def posted_create_tx(b, signed_create_tx):
```

Comment:
Is this a static fixture?

Reply:
Nope, good catch - added an `abci` mark.